product/reports/670_Performance by Asset Type - Middleware Servers/130_JVM Garbage Collection - daily over last week.yaml

```diff
@@ -0,0 +1,53 @@
+--- 
+where_clause: 
+dims: 
+created_on: 2016-05-23 14:53:10.822035 Z
+reserved: 
+title: "JVM Garbage Collection - daily averages for last week"
+conditions: 
+updated_on: 2016-05-23 00:35:33.732482 Z
+order: Descending
+graph: 
+menu_name: "JVM Garbage Collection - daily over last week"
+rpt_group: Custom
+priority: 
+col_order: 
+- id
+- name
+- start_time
+- mw_accumulated_gc_duration_min
+- mw_accumulated_gc_duration_avg
+- mw_accumulated_gc_duration_max
+timeline: 
+id: 99
```

Comment:
In the reports folder, the yaml file should provide its own id.
Thanks for the catch, reviewing it, I notice I reused the same id on the three reports where in other folders this id is different.